CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [1.9.0] - 2023-09-11
```

Comment:
Maybe the date should be 11-09?

Reply:
Sorry maybe 11-08, as other repos are 11-08

Reply:
ahh I thought so, I got confused in the format by looking at the one below :P